rpc/tests/test_seth_rpc.py

```diff
@@ -80,6 +80,7 @@ def setUpClass(cls):
         # account values
         cls.public_key = "036d7bb6ca0fd581eb037e91042320af97508003264f08545a9db134df215f373e"
         cls.account_address = "434d46456b6973a678b77382fca0252629f4389f"
+        cls.account_address_b = bytes([0x43, 0x4d, 0x46, 0x45, 0x6b, 0x69, 0x73, 0xa6, 0x78, 0xb7, 0x73, 0x82, 0xfc, 0xa0, 0x25, 0x26, 0x29, 0xf4, 0x38, 0x9f])
```

Comment:
Might be worth writing this as `cls.account_address_b = bytes.fromhex(cls.account_address)` so that the two definitions don't get out of sync

Reply:
good point - I missed that helper function - will fix.